x-pack/test/alerting_api_integration/security_and_spaces/tests/alerting/delete.ts

```diff
@@ -301,6 +301,9 @@ export default function createDeleteTests({ getService }: FtrProviderContext) {
             .send(getTestAlertData())
             .expect(200);
 
+          // Delay before performing update to avoid 409 errors
+          await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
I wonder if we should wrap the call below in a `retry.try` instead of adding timeouts in the tests? That way it can keep trying for a certain amount of time but in most cases finish the test at normal speed. Unless this was a consistent issue?

Reply:
> I wonder if we should wrap the call below in a `retry.try` instead of adding timeouts in the tests? That way it can keep trying for a certain amount of time but in most cases finish the test at normal speed. Unless this was a consistent issue?

Good call. The failures don't happen very often. I've updated the PR to use `retry.try`. Here are the flaky test runner results after that update: https://kibana-ci.elastic.co/job/kibana+flaky-test-suite-runner/1039/